### DISCUSSION TITLE
p-select dropdown position -> how to display the dropdown above the input element?

### DISCUSSION THREAD

**Discussion by sab9900** [primefaces/community#2944] [PrimeNG]
HI there!

My designers decided to have the paginator at the bottom of the list. now there is no space to open the p-select dropdown to change the visible Rows. 
How can i set the dropdown position to be above the input element?
i can't be the only one having this issue right?

best regards and thank you in advance!

**Comment by faheemhameed** [comment]
I am having the same issue. I also have the paginator for my list page at the bottom of my page.
Also, I am having the same issue in my advanced search modal dialog.

@cagataycivici  I need your kind help on this.

We are using: primeng v18 beta 3

A million thanks!!!

Regards,
Hameed

![Screenshot 2024-11-03 at 11 26 18 AM](https://github.com/user-attachments/assets/bc417ba4-95bd-4d08-aa44-f6789c845038)

![Screenshot 2024-11-03 at 11 25 56 AM](https://github.com/user-attachments/assets/2bc19c5e-8086-4863-8a2a-8bdf969c8550)

**Comment by abennouna** [comment]
Have you tried using `appendTo`, like in previous PrimeNG versions?

- In Paginator --> https://v18.primeng.org/paginator#api.paginator.props.appendTo
- In Select --> https://v18.primeng.org/select#api.select.props.appendTo

**Comment by faheemhameed** [comment]
It didn't work well. We have tried this multiple times.

Finally, we used onShow and onFilter callbacks to do some ugly fixes to calculate the top position of the select dropdown's overlay panel.

I am still waiting for this to be fixed so that I can remove my unnecessary fixes.

Thanks!!
